packages/playwright-core/src/server/bidi/third_party/firefoxPrefs.ts

```diff
@@ -265,6 +265,9 @@ function defaultProfilePreferences(
 
     // Prevent starting into safe mode after application crashes
     'toolkit.startup.max_resumed_crashes': -1,
+
+    // Allow localhost and 127.0.0.1 to be proxied
+    'network.proxy.allow_hijacking_localhost': true,
```

Comment:
These preferences are copied from https://github.com/puppeteer/puppeteer/blob/main/packages/browsers/src/browser-data/firefox.ts as the header of this file mentions. I don't see this setting upstream, why do we need it here? If it is just for the tests, you can pass it via `firefoxUserPrefs`

Reply:
I would assume it's for testing only. So the suggestion would make sense.

Also it's not a pref that we enable by default in Firefox so we shouldn't do it in general for Playwright as well.